lib/virt_autotest/virtual_network_utils.pm

```diff
@@ -44,7 +44,9 @@ our @EXPORT
 sub check_guest_module {
     my ($guest, %args) = @_;
     my $module = $args{module};
+    my $net    = $args{net} // "br123";
     if (($guest =~ m/sles-?11/i) && ($module eq "acpiphp")) {
+        save_guest_ip("$guest", name => $net);
```

Comment:
Good point! 

Because of it'd better to make sure connection both vm host and guest system together before check guest loaded kernel modules. 

Also, refer to the follow situations:
1.  Depended on routed virtual network required, there was cloned new guest from existed guest system, named as {guest}.clone, which was not saved in file /etc/hosts before. So before check this cloned guest system about loaded kernel module, it'd better call save_guest func here to make sure the connection both vm host and this cloned guest system together
2. Depended on virtual network required on vm host without host bridge as default, there was be changed vm host network setting to shut down the existed guest and create host bridge interface, then boot up the existed guest again - Guest system IP address will be changed depended on vm host network setting changed. So, before check guest loaded kernel module, we'd better to call save_guest func here to make sure the connection both vm host and guest system together here

Thus,  depended on the involution during virtual network testing, we'd better to call save_guest func firstly before check guest loaded kernel modules.

Reply:
The guest should have the hosts's ssh key in `~/.ssh/authorized_keys` before it is cloned.

Reply:
Great, we have the same idea. 

For now,  The guest system have the vm host ssh key was finished at `tests/virt_autotest/libvirt_virtual_network_init` already.